Library/Homebrew/cmd/reinstall.rb

```diff
@@ -56,7 +57,8 @@ def reinstall
 
     Install.perform_preinstall_checks
 
-    args.resolved_formulae.each do |f|
+    resolved_formulae, possible_casks = args.resolved_formulae_and_unknowns
```

Comment:
I wonder if this could be `resolved_formulae_casks_unknowns` and split them into three separate lists so there's no loading/parsing needed here?